### DISCUSSION TITLE
Recently viewed/edited files

### DISCUSSION THREAD

**Discussion by tomzx** [damionx7/Zettel-Notes-Documentation#88] [Feature Requests]
Like most text editor, it would be great if you could see a list of recently viewed or edited files in order to quickly go back to any of those files.

**Comment by aamott** [comment]
You can currently select the button between the search and kebab menu to sort by (says filter by, but it means sort by) many things, including last modified or accessed. You can also choose notes first or folders first, and to reverse the list. Is that what you're going after?

**Comment by tomzx** [comment]
The edited files can indeed be found in the calendar view, the recently viewed files cannot. It would require keeping this information available in a database, or if the file system supports it, use the "accessed" timestamp and introduce some way to view the calendar by this feature + search/filter by it.

**Reply by damionx7** [reply]
I have added this feature https://t.me/zettelnotes/342. It will be live in the next update.
